### ISSUE TITLE
Automated smoke test: css tests fails with open file dialog 

### ISSUE THREAD

**Issue by aeschli** [Microsoft/vscode#27930]
Linux Elementary OS Freya running in Parallels.

Version 1.13.0-insider
Commit af2752dad442341000b0bf95861b91ecd8263ca0
Date 2017-06-02T05:12:05.664Z

Running the automated test I get the following failures:
```
1) Smoke Test Suite CSS verifies quick outline:

      AssertionError: 0 == 2
      + expected - actual

      -0
      +2
      
      at Context.<anonymous> (out/tests/css.js:43:24)
      at Generator.next (<anonymous>)
      at fulfilled (out/tests/css.js:8:58)
      at process._tickDomainCallback (internal/process/next_tick.js:129:7)


  2) Smoke Test Suite CSS verifies warnings for the empty rule:
     AssertionError: undefined == true
      at Context.<anonymous> (out/tests/css.js:55:24)
      at Generator.next (<anonymous>)
      at fulfilled (out/tests/css.js:8:58)
      at process._tickDomainCallback (internal/process/next_tick.js:129:7)


  3) Smoke Test Suite CSS verifies that warning becomes an error once setting changed:
     AssertionError: undefined == true
      at Context.<anonymous> (out/tests/css.js:68:24)
      at Generator.next (<anonymous>)
      at fulfilled (out/tests/css.js:8:58)
      at process._tickDomainCallback (internal/process/next_tick.js:129:7)


  4) Smoke Test Suite JavaScript shows correct quick outline:

      AssertionError: 0 == 12
      + expected - actual

      -0
      +12
      
      at Context.<anonymous> (out/tests/javascript.js:43:24)
      at Generator.next (<anonymous>)
      at fulfilled (out/tests/javascript.js:8:58)
      at process._tickDomainCallback (internal/process/next_tick.js:129:7)


  5) Smoke Test Suite Tasks verifies build errors are reflected in 'Problems View':
     Error: the string "Could not retrieve the element in 45 seconds." was thrown, throw an Error :)
```

**Comment by aeschli** [comment]
![css-smoketest](https://cloud.githubusercontent.com/assets/6461412/26731755/55c35218-47b5-11e7-8ac3-3fa6ebefc942.gif)

**Comment by michelkaporin** [comment]
I have tested it with Elementary OS Loki and Ubuntu running in Parallels Desktop, neither results in the same issue you are hitting.

I have also tried uninstalling the following apps on your VM image one by one and it did not result in correct behaviour:
```
Keyboard Input Methods
Input Method (im-config)
Orca Screen Reader
```

I suspect that Elementary OS Freya is not friendly with the way ChromeDriver is triggering some of the keys, that is why you are hitting Open Recent menu. There are similar issues raised for ChromeDriver, when the keys interpreted by it were not the right ones in a particular setup (most people were SSHing with X11). So it could be also a problem on ChromeDriver or Selenium side.

Hence, suggesting you to upgrade your OS to the Loki and see if it works 🙂 .
